docs_src/response_model/tutorial006.py

```diff
@@ -1,39 +1,74 @@
-from typing import Optional
+from typing import List, Optional
 
 from fastapi import FastAPI
-from pydantic import BaseModel
+from pydantic import BaseModel, HttpUrl
 
 app = FastAPI()
 
 
+class Image(BaseModel):
+    url: HttpUrl
+    name: str
+
+
+class Video(BaseModel):
+    url: HttpUrl
+    duration: int
+
+
 class Item(BaseModel):
     name: str
-    description: Optional[str] = None
-    price: float
-    tax: float = 10.5
+    image: Optional[Image] = None
+    videos: Optional[List[Video]] = None
 
 
 items = {
-    "foo": {"name": "Foo", "price": 50.2},
-    "bar": {"name": "Bar", "description": "The Bar fighters", "price": 62, "tax": 20.2},
-    "baz": {
-        "name": "Baz",
-        "description": "There goes my baz",
-        "price": 50.2,
-        "tax": 10.5,
+    "foo": {
+        "name": "Foo",
+        "image": {"url": "http://test.com/img.jpg", "name": "Some image"},
+        "videos": [
+            {"url": "http://test.com/video_1.mp4", "duration": 40},
+            {"url": "http://test.com/video_2.mp4", "duration": 120},
```

Comment:
For `"url"` use https://example.com instead of https://test.com since https://example.com is a reserved domain meant for examples in documentation, etc. 

For more information please visit: https://www.iana.org/domains/reserved

Reply:
fixed that